packages/mint-components/src/components/tax-and-cash/sqm-tax-and-cash/sqm-tax-and-cash.tsx

```diff
@@ -0,0 +1,1041 @@
+import { UseTaxAndCashResultType, useTaxAndCash } from "./useTaxAndCash";
+import { extractProps } from "./extractProps";
+import { isDemo } from "@saasquatch/component-boilerplate";
+import LoadingView from "./LoadingView";
+
+/**
+ * @uiName Tax and Cash
+ * @exampleGroup Tax and Cash
+ * @example Tax and Cash Multi Step Form - <sqm-tax-and-cash></sqm-tax-and-cash>
+ */
+@Component({
+  tag: "sqm-tax-and-cash",
+  shadow: true,
+})
+export class TaxAndCashMonolith {
+  // !: Make sure to also change the prop names in the respective component
+
+  /**
+   * * USER INFO FORM PROPS:
+   **/
+
+  /**
+   * Label text for first name input
+   * @uiName First name label
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_firstName: string = "First name";
+  /**
+   * Label text for last name input
+   * @uiName Last name label
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_lastName: string = "Last name";
+  /**
+   * Label text for email input
+   * @uiName Email label
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_email: string = "Email";
+  /**
+   * Label text for country input
+   * @uiName Country label
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_country: string = "Country";
+  /**
+   * Label text for currency input
+   * @uiName Currency label
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_currency: string = "Currency";
+  /**
+   * Label text for tax and banking collection checkbox
+   * @uiName Tax and banking label
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_allowBankingCollection: string = "I agree to the terms";
+  /**
+   * Heading text for the tax and banking collection checkbox
+   * @uiName Tax and banking collection checkbox heading
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_taxAndBankingCollection: string = "Continue";
+  /**
+   * Sub text shown at the top of the page, used to show the current step of the tax form.
+   * @uiName Tax form step text
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_formStep: string = "Step 1 of 4";
+  /**
+   * Heading text shown above the forms inputs.
+   * @uiName Form heading text
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_personalInformation: string = "Personal Information";
+  /**
+   * Label text for the business entity radio button
+   * @uiName Business entity radio button label
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_businessEntity: string = "I represent a business entity";
+  /**
+   * Label text for the individual participant radio button
+   * @uiName Individual participant radio button label
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_individualParticipant: string =
+    "I am an individual participant";
+  /**
+   * Heading text for the participant type radio buttons
+   * @uiName Participant type radio buttons heading
+   * @uiGroup Step 1 Properties
+   */
+  @Prop() step1_participantType: string = "Participant type";
```

Comment:
I think this can be removed since its not used in step 1 anymore? There might be some others that can be removed too